### ISSUE TITLE
[craftercms] MySQL fails to stop consistently in windows

### ISSUE THREAD

**Issue by russdanner** [craftercms/craftercms#1630]
### Expected behavior
Gradlew stop and batch files used to stop studio should _always_ stop MySQL

### Actual behavior
Mysql should always stop when stopping studio.
   When Mysql does not stop, subsequent start-up activities fail because MySQL is already running.
Restart should work consistently.

### Steps to reproduce the problem
* start studio on windows 
* stop it 
* note mysql does not always die

(applies to both Gradle and Batch Files)

### Log/stack trace (use https://gist.github.com)
NA

### Specs: version, OS, browser, etc.
Windows

**Comment by cortiz** [comment]
Quick update, 
Scripts do shutdown MariaDB every time, issue **does happen** when the tomcat window is close,
this applies to Linux if the tomcat process is `kill -9`

**Comment by sumerjabri** [comment]
@cortiz I think I know what's happening:
- Tomcat doesn't shut down correctly on Windows or takes too long
- `crafter.bat` shuts down Tomcat before it had a chance to shut down MariaDB

@cortiz let's do two things:
1. Validate that Tomcat doesn't shut down and fix that if possible
2. Check for MariaDB port during startup and refuse to start if the port is in use and show the message:
``` 
Crafter CMS Database Port: {port} is in use by process {process_id}. This might be because of a prior unsuccessful or incomplete shut down. Please terminate that process before attempting to start Crafter CMS.
```

If we can't fix #1, we can discuss alternatives.

**Comment by cortiz** [comment]
@sumerjabri on #1640 , 

Now we wait about 5 sec after tomcat shutdown is down and check that MariaDB isn't down will gracefully shut it down (or killed if been nice does not work)

Added the startup message
